src/models/data.py

```diff
@@ -124,6 +128,18 @@ def __init__(self, data_path: Path = Path('data'), batch_file_size: int = 1,
         self.to_tensor = to_tensor
         self.ignore_vars = ignore_vars
 
+        self.static = None
+        self.static_normalizing_dict = None
+
+        if static:
+            static_data_path = data_path / 'features/static/data.nc'
```

Comment:
so static is always written to one `data.nc` file?

Reply:
yup!